src/osa/scripts/closer.py

```diff
@@ -175,20 +176,24 @@ def post_process(seq_tuple):
     if not options.no_dl2:
         merge_files(seq_list, data_level="DL2")
 
-    time.sleep(300)
+    time.sleep(600)
 
     # Check if all jobs launched by autocloser finished correctly 
     # before creating the NightFinished.txt file
-    n_max = 10
+    n_max = 6
     n = 0
     while not all_closer_jobs_finished_correctly() & n <= n_max:
         log.info(
             "All jobs launched by autocloser did not finished correctly yet. "
-            "Checking again in 5 minutes..."
+            "Checking again in 10 minutes..."
         )
-        time.sleep(300)
+        time.sleep(600)
         n += 1
 
+    if n > n_max:
+        send_warning_mail(date=options.date)
```

Comment:
the problem with this way of sending mail is that in the current scheme, in which autocloser is launched by a cron job twice per hour for the whole day, it will keep sending warning mails. In this sense, a unique cronjob checking the night is finished night once per day, and if not sending an email, could be a better solution.

Also since now running autocloser could take up to 1 hour, the frequency of the cron job has to be updated correspondingly.